src-gui/src/main/java/org/openkilda/utility/SwitchUtil.java

```diff
@@ -16,7 +16,7 @@
 package org.openkilda.utility;
 
 import org.apache.commons.collections4.MapUtils;
-import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang.StringUtils;
```

Comment:
Why downgrading? lang3 is a more modern version of the commons.

Reply:
nice catch, replaced in all the places